woocommerce/woocommerce
# Add create product actions in products data store

### All Submissions:






### Changes proposed in this Pull Request:

Adds the ability to create products via the products data store.

Closes (part of) #33076 .

### How to test the changes in this Pull Request:

1. Open your browser's console
2. Run this command (with any product data)
```
wp.data.dispatch('wc/admin/products').createProduct({
    name: 'Donkey Sauce',
    regular_price: 20.18,
    description: 'Take a trip to Flavortown!',
});
```
3. Check that the product was added on the Products page

### Other information:




### FOR PR REVIEWER ONLY:
